### ISSUE TITLE
Displays error when open

### ISSUE THREAD

**Issue by atreyv** [spyder-ide/spyder#7683]
### What steps will reproduce the problem?

Installed with conda in miniconda environment, ubuntu 16.04.5.
Run spyder in terminal, conda environment.

### Traceback
```python-traceback
  File "/home/pedro/dev/conda-python/envs/qutip-env/lib/python3.6/site-packages/spyder/app/mainwindow.py", line 1961, in valid_project
    if bool(self.projects.get_active_project_path()):
AttributeError: 'NoneType' object has no attribute 'get_active_project_path'
```

## Versions

* Spyder version: 3.3.0 
* Python version: 3.6.6
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Linux 4.15.0-30-generic

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
sphinx >=0.6.6    :  1.7.6 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.12.1 (OK)
psutil >=0.3      :  5.4.6 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
pandas >=0.13.1   :  None (NOK)
numpy >=1.7       :  1.15.0 (OK)
sympy >=0.7.3     :  1.2.1.dev (OK)
cython >=0.21     :  0.28.5 (OK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0     :  6.5.0 (OK)
matplotlib >=2.0.0:  2.2.2 (OK)
pylint >=0.25     :  2.1.1 (OK)
```

**Comment by ccordoba12** [comment]
This is a duplicate of issue #7403 and it was fixed in our 3.3.1 release. If you're using Anaconda, please be patient, new conda packages for that version should be available shortly.

**Comment by atreyv** [comment]
Thanks for the update!
